lib/redix/uri.ex

```diff
@@ -38,6 +38,8 @@ defmodule Redix.URI do
   end
 
   defp database(%URI{path: "/" <> path = full_path}) do
+    path = path |> String.split("/") |> hd()
```

Comment:
I think instead of doing this you can just avoid matching on `""` on line 43 below, and instead match on `{db, "/" <> _}`. Right?

Reply:
Good catch! Fixed!